### PR TITLE
GUI bash_completion conflict with loguru verbosity switch -v

### PR DESCRIPTION
**Pull request by arrowgent**
`dosbox -v` currently controls loguru verbosity
its not defined for equivalent `--version`

### CODE REVIEW

**Comment by arrowgent** [comment]
https://github.com/dosbox-staging/dosbox-staging/pull/1475

**Review by kcgen — CHANGES_REQUESTED** [review]
Can you PR this to the dosbox-staging/loguru repo?

This is "upstream", and will make sure this change is not lost when we copy over new updates from that repo.

It will also let us push that change further upstream to the original repo for the author to consider.

File: src/gui/gui_msgs.h [hunk 1]
```diff
@@ -72,7 +72,7 @@ These are common options:
   -exit               Dosbox will close itself when the DOS program
                       specified by FILE ends.
 
-  -v, --version       Output version information and exit.
+  --version       Output version information and exit.
```
Comment (kcgen) [inline src/gui/gui_msgs.h:75]: Looks good :+1:

**Review by kcgen — CHANGES_REQUESTED** [review]
Thanks @arrowgent !

Only two changes - please move the Loguru commit to that repo (https://github.com/dosbox-staging/loguru), and remove that commit this this PR.  Reason is, we copy from that upstream repo and we don't edit our local file here.. it's overwritten/clobbered by the upstream.

Second: please update the commit messages to follow "the imperative" style, or "command style" phrasing. Tips here:   https://github.com/dosbox-staging/dosbox-staging/blob/main/CONTRIBUTING.md#commit-messages

You can see

**Comment by arrowgent** [comment]
you have no idea how difficult it is to find any way to revert a commit in the browser of github
or a way to remove a commit from this PR ha

**Comment by arrowgent** [comment]
madness.  some simple features through a browser are limited to other tools (cli)
https://github.blog/2014-06-24-introducing-the-revert-button/

closing and remaking the PR to get rid of my fiddling around

**Comment by kcgen** [comment]
I'm very sorry for putting you through that. I didn't realize you were using the web UI. 
I am very happy to help on my side and still attribute the commits to you - if that works. Just shout.

**Comment by arrowgent** [comment]
no worries.  i should get onboard with using my terminal skills for managing git one of these days!

**Comment by Wengier** [comment]
I used the web UI in the beginning too, before getting to use Git command.

**Comment by Grounded0** [comment]
https://www.sublimemerge.com/

My choice.

**Pull request closed without merging**